modules/ascat/main.nf

```diff
@@ -85,16 +87,16 @@ process ASCAT {
 
     #Load the data
     ascat.bc = ascat.loadData(
-        Tumor_LogR_file = "Tumour_tumourLogR.txt",
-        Tumor_BAF_file = "Tumour_normalBAF.txt",
-        Germline_LogR_file = "Tumour_normalLogR.txt",
-        Germline_BAF_file = "Tumour_normalBAF.txt",
+        Tumor_LogR_file = paste0("$prefix", "_tumour_tumourLogR.txt"),
+        Tumor_BAF_file = paste0("$prefix", "_tumour_tumourBAF.txt"),
+        Germline_LogR_file = paste0("$prefix", "_tumour_normalLogR.txt"),
+        Germline_BAF_file = paste0("$prefix", "_tumour_normalBAF.txt"),
         genomeVersion = "$genomeVersion",
         gender = "$gender"
     )
 
     #Plot the raw data
-    ascat.plotRawData(ascat.bc, img.prefix = "Before_correction_")
+    ascat.plotRawData(ascat.bc, img.prefix = paste0("$prefix", "_before_correction_"))
```

Comment:
for readablity, I actually think the `.` is easier 🙈

Reply:
okay 👍